src/components/Panels/Telescope/Targets/TargetSwapButton.tsx

```diff
@@ -29,40 +39,57 @@ export function TargetSwapButton({ selectedTarget }: { selectedTarget: Target |
   const { data: instrumentData, loading: instrumentLoading } = useInstrument({
     variables: { name: configuration?.obsInstrument ?? '', issPort: 3, wfs: 'NONE' },
   });
+
+  const { data: acData, loading: acLoading } = useInstrument({
+    variables: { name: `AC_${configuration?.site}` },
+  });
+
+  const acInst = acData?.instrument;
   const instrument = instrumentData?.instrument;
   const { data: rotatorData, loading: rotatorLoading } = useRotator();
   const rotator = rotatorData?.rotator;
 
   const loading =
-    stateLoading || swapLoading || restoreLoading || instrumentLoading || rotatorLoading || configurationLoading;
+    stateLoading ||
+    swapLoading ||
+    restoreLoading ||
+    instrumentLoading ||
+    rotatorLoading ||
+    configurationLoading ||
+    acLoading;
 
   const disabled = !canEdit;
 
   const label = data?.onSwappedTarget ? 'Point to Base' : 'Point to Guide Star';
 
   const onClick = () => {
-    if (selectedTarget?.id && instrument && rotator) {
+    if (selectedTarget?.id && instrument && rotator && oiSelected && acInst) {
```

Comment:
Can you add text for these new checks in the details toast below? Other than that LGTM

Reply:
Sure, I will do